src/OpenTelemetry/Metrics/PeriodicExportingMetricReader.cs

```diff
@@ -22,20 +22,31 @@ namespace OpenTelemetry.Metrics
 {
     public class PeriodicExportingMetricReader : BaseExportingMetricReader
     {
+        internal const int DefaultExportIntervalMilliseconds = 60000;
+        internal const int DefaultExportTimeoutMilliseconds = 30000;
+
         private readonly Task exportTask;
         private readonly CancellationTokenSource token;
 
-        public PeriodicExportingMetricReader(BaseExporter<Metric> exporter, int exportIntervalMs)
+        public PeriodicExportingMetricReader(
+            BaseExporter<Metric> exporter,
+            int exportIntervalMilliseconds = DefaultExportIntervalMilliseconds,
+            int exportTimeoutMilliseconds = DefaultExportTimeoutMilliseconds)
```

Comment:
Want to get feedback on the naming - for `BatchExportProcessor` we call it `exporterTimeoutMilliseconds` rather than `exportTimeoutMilliseconds`.

I think it's unlikely that we'll change `exportIntervalMilliseconds` to `exporterIntervalMilliseconds`, so do we try to align here?

Reply:
I would vote for `exportTimeoutMilliseconds` & `exportIntervalMilliseconds` over `exporterTimeoutMilliseconds` & `exporterIntervalMilliseconds`. "ExporterTimeoutMilliseconds" to me seems to indicate "this timeout will be used anywhere the exporter needs a timeout" rather than "the timeout which will be applied to the export function." But probably a minor thing, we could also clarify with comments/XML docs.